frontend/src/e2e-playwright/pages/citizen/citizen-messages.ts

```diff
@@ -0,0 +1,20 @@
+import { RawElement, RawTextInput } from 'e2e-playwright/utils/element'
+import { Page } from 'playwright'
+
+export default class CitizenMessagesPage {
+  constructor(private readonly page: Page) {}
+
+  #messageReplyContent = new RawTextInput(
+    this.page,
+    '[data-qa="message-reply-content"]'
+  )
+  #threadListItem = new RawElement(this.page, '[data-qa="thread-list-item"]')
+  #sendReplyButton = new RawElement(this.page, '[data-qa="message-send-btn"]')
+
+  async replyToThread(index: number, content: string) {
+    await this.#threadListItem.click()
+    await this.#messageReplyContent.fill(content)
+    await this.#sendReplyButton.click()
+    await this.page.waitForTimeout(1000)
```

Comment:
In general it's not a good idea to just do a fixed time timeout. If you need to wait for some async action to complete, read the UI for an indication that it completed.